### PR TITLE
Adds @license JSDoc tag to license comments

### PR DESCRIPTION
**Pull request by nathanmosher**
Hi, just hoping to get this license tag submitted upstream to help this code play a bit better with the NodeJS import tools internally (at Google). The change will help a Google3 import script know that a comment block is, in fact, a license comment block.

There's a few trailing whitespaces my editor really wanted to remove too, lmk if I should revert them, and if you need more context.

### CODE REVIEW

**Review by lahirumaramba — APPROVED** [review]
LGTM! Thank you!

**Comment by hiranya911** [comment]
@lahirumaramba please merge this when ready. I don't think @nathanmosher has write access on this repo.

**Comment by lahirumaramba** [comment]
> @lahirumaramba please merge this when ready. I don't think @nathanmosher has write access on this repo.

Sounds good! I started the process.

**Pull request merged**
